charts/cronjob-aws-ocp-snap/values.yaml

```diff
@@ -1,12 +1,12 @@
-aws_access_key_id: '# TODO: must define a default value for .aws_access_key_id'
-aws_region: '# TODO: must define a default value for .aws_region'
-aws_secret_access_key: '# TODO: must define a default value for .aws_secret_access_key'
+aws_access_key_id: changeme
```

Comment:
Isn't it better to make this something that will error out to avoid having invalid values (i.e.: `changeme`) be deployed?

Reply:
@oybed yeah, I was trying to figure out a way to quickly go through them all and do a `--dry-run` install since I discovered that a bunch of the templates were failing local schema validation. But I can set them back if you'd prefer.

Reply:
I'm good with leaving, but perhaps make it something a bit clearer that it needs to be replaced, something like: `PLEASE_CHANGE_ME_WITH_VALID_VALUE` ... I believe that will still be valid and lets the dry-run pass, but catches the eyes a bit more. + perhaps add a README at the `charts` level to call this out?